static/sass/styles.scss

```diff
@@ -494,4 +494,45 @@ $p-small-lh-diff: map-get($line-heights, default-text) - map-get($line-heights,
 /// can be removed once added upstream
 .is-paper {
   background-color: #f3f3f3;
-}
\ No newline at end of file
+}
+
+.p-department-cards__container {  
+  aspect-ratio: 2 / 3;
+  background-color: #fff;
+  margin-bottom: $spv--large;
+  margin-top: $spv--small;
+
+  @media screen and (max-width: ($breakpoint-large - 1)) {
+    aspect-ratio: auto;
+  }
+
+  .p-department-cards__card {
+    @extend %vf-bg--x-light;
+    @extend %vf-card-padding;
+    // Push roles text down
+    @extend .u-stack;
+
+    overflow: auto;
+    padding: $spv--large 1.5rem;
+
+    @media screen and (max-width: ($breakpoint-large - 1)) {
+      align-items: baseline;
```

Comment:
Same as above, ideally make it mobile first, and override anything of larger breakpoints.